MoltenVK/MoltenVK/API/mvk_config.h

```diff
@@ -191,7 +191,7 @@ typedef struct {
 	 * and the changed value will immediately effect subsequent MoltenVK behaviour.
 	 *
 	 * The initial value or this parameter is set by the
-	 * MVK_DEBUG
+	 * MVK_CONFIG_DEBUG
```

Comment:
This is a change in MoltenVK's external interface, and it should probably be documented somewhere.

Reply:
Good catch! Thanks for noticing this. I didn't fully deal with the legacy part of that change.

I've added additional documentation about the change, and some env var code to continue to accept `MVK_DEBUG` as a legacy env var.